modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/setupTests.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {configure} from 'enzyme';
```

Comment:
I've seen in previous pull requests that you were using Enzime.js, and wanted to share [this](https://www.piotrstaniow.pl/goodbye-enzyme) link with you just in case. Is there any reason why we can't use [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) like we usually do in dxp?

Reply:
Sorry @julien i missed it! But we're discussing about it

Reply:
No problem at all, it was just a "heads up"